src/utils.js

```diff
@@ -13,10 +13,10 @@ export function randomItem(arrayOfItems){
   return arrayOfItems[Math.floor(Math.random() * arrayOfItems.length)];
 }
 
-export function randomString(length) {
-  const charset = 'abcdefghijklmnopqrstuvwxyz';
+export function randomString(length, charset='') {
+  if (!charset) charset = 'abcdefghijklmnopqrstuvwxyz';
```

Comment:
You can just set the default value directly in the function declaration.
```suggestion
export function randomString(length, charset='abcdefghijklmnopqrstuvwxyz') {
```

Reply:
Sure - that checks out fine - I actually had pulled the example from Tom's Slack reply.

Also, much credit to him for many of the test cases also.